packages/jira-adapter/test/filters/forms/forms.test.ts

```
@@ -591,7 +570,7 @@ describe('forms filter', () => {
     beforeEach(async () => {
       const config = _.cloneDeep(getDefaultConfig({ isDataCenter: false }))
       config.fetch.enableJSM = true
-      const { client: cli, connection: conn } = mockClient(false)
+      const { client: cli, connection: conn } = mockClient(false, DEFAULT_CLOUD_ID)
```

Here you left DEFAULT_CLOUD_ID
I think I am missing something- you removed the default value but did not add it as default to the mockClient code

---

I don't think I understand what you mean 😅